### DISCUSSION TITLE
Safest time to jump to V2

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#867] [Q&A]
I'm planning to get my team on V2 as soon as it supported by the latest shipping version of Visual Studio and .Net 5.  I don't want folks on the RC versions of things.  My guess is Tuesday 11/10 MS will drop .Net 5 and an updated version of Visual Studio to go with it.  So, 11/11 would be an appropriate time for us to jump with the tooling.

Is Oqtane also planning a day-one release of V2?  So planning for 11/11 is good timing to jump here as well?

**Comment by sbwalker** [comment]
Yes, the plan is to release V2..0 on 11/11

**Reply by zhengdayongbob** [reply]
Great!

**Comment by sbwalker** [comment]
Oqtane 2.0 was published yesterday:

https://github.com/oqtane/oqtane.framework/releases/tag/v2.0.0

https://www.oqtane.org/Resources/Blog/PostId/544/announcing-oqtane-20-for-net-5

My .NET Conf session is later today
